vms/rpcchainvm/vm_client.go

```diff
@@ -802,6 +802,33 @@ func (vm *VMClient) GetStateSummary(ctx context.Context, summaryHeight uint64) (
 	}, err
 }
 
+func (vm *VMClient) BackfillBlocksEnabled(ctx context.Context) (ids.ID, uint64, error) {
+	resp, err := vm.client.BackfillBlocksEnabled(ctx, &emptypb.Empty{})
+	if err != nil {
+		return ids.Empty, 0, err
+	}
+	if errEnum := resp.Err; errEnum != vmpb.Error_ERROR_UNSPECIFIED {
+		return ids.ID(resp.Id), resp.Height, errEnumToError[errEnum]
+	}
+	return ids.ID(resp.Id), resp.Height, nil
+}
+
+func (vm *VMClient) BackfillBlocks(ctx context.Context, blocks [][]byte) (ids.ID, uint64, error) {
+	resp, err := vm.client.BackfillBlocks(
+		ctx,
+		&vmpb.BackfillBlocksRequest{
+			BlksBytes: blocks,
+		},
+	)
+	if err != nil {
+		return ids.Empty, 0, err
+	}
+	if errEnum := resp.Err; errEnum != vmpb.Error_ERROR_UNSPECIFIED {
+		return ids.ID(resp.Id), resp.Height, errEnumToError[errEnum]
+	}
+	return ids.ID(resp.Id), resp.Height, nil
+}
```

Comment:
this is all we need to extend the interface in the rpcchainvm client, so I figured I coded it in this first PR to reduce diffs down the line